xml_sitemap_writer.py

```diff
@@ -48,30 +76,47 @@ def __init__(self, path: str, root_url: str):
 
         self.add_section("pages")
 
-    def add_url(self, url: str):
-        """
-        Add a given URL to the sitemap
-        """
-        # lazily create a new sub-sitemap file
-        # see add_section() method
+
+    def add_url(
+        self,
+        url: str,
+        lastmod: Optional[str] = None,
+        priority: Optional[str] = None,
+        changefreq: Optional[str] = None,
+    ):
         if self.sitemap_urls_counter == 0:
             self._add_sitemap()
 
         self.total_urls_counter += 1
         self.sitemap_urls_counter += 1
 
-        # check per sitemap limits
         if self.sitemap_urls_counter > self.URLS_PER_FILE:
-            self.logger.info(
-                f"URLs per sitemap counter reached the limit of {self.URLS_PER_FILE}"
-            )
+            self.logger.info(f"URLs per sitemap counter reached the limit of {self.URLS_PER_FILE}")
```

Comment:
```suggestion
            self.logger.info(
                f"URLs per sitemap counter reached the limit of {self.URLS_PER_FILE}"
            )
```

Bring back a properly formatted line.